components/breadcrumb-item.js

```diff
@@ -0,0 +1,30 @@
+const template = `
+<li class="c-breadcrumb__item" :class="classes">
+    <a v-if="!current" :href="href">
+      <slot></slot>
+    </a> 
+    <slot v-else></slot>
```

Comment:
Can you use `v-if` and `v-else` without using `<template>`?

Reply:
I miss to see where am I using `<template>` ?